README.md

```diff
@@ -77,6 +77,22 @@ Sample query:
 }
 ```
 
+Another sample query:
+```graphql
+query {
+  organizationMetas(
+    organizationNameILike: "%foundation%"
+    orderByMulti: [["totalFunded", "DESC NULLS LAST"]]
```

Comment:
Does sequelize sanitize this for us? Googling around I can't find a definitive answer since this seems to blend sequelize and graphql & there's not a lot of info on `graphql-sequelize`

Reply:
good quesiton; it's passed to the sequelize model.findAll options, which, now that i'm looking into it, only sanitizes the column but not the args - i'm going to patch this so that we only allow `DESC` and `ASC` and always mutate these into `DESC NULLS LAST` and `ASC NULLS LAST`

Reply:
Good stuff! That'll stop those Russians.